packages/rspack/src/loader-runner/index.ts

```diff
@@ -1020,11 +1048,15 @@ export async function runLoaders(
 					}
 					if (!fn) continue;
 
-					const args = await isomorphoicRun(fn, [
-						loaderContext.remainingRequest,
-						loaderContext.previousRequest,
-						currentLoaderObject.loaderItem.data
-					]);
+					const args = await isomorphoicRun(
```

Comment:
@h-a-n-a I don't follow this function, it seems the args passed here is very different with the following(remainingRequest vs content